quinn-proto/src/endpoint.rs

```diff
@@ -55,18 +54,14 @@ pub struct Endpoint {
 }
 
 impl Endpoint {
-    pub fn new(
-        log: Logger,
-        config: Config,
-        server_config: Option<ServerConfig>,
-    ) -> Result<Self, EndpointError> {
+    pub fn new(log: Logger, config: Config, server_config: Option<ServerConfig>) -> Self {
```

Comment:
#210 just inspired me to put some `Config` sanity-checking in here, though I'm not sure if we should use panics or `Result` to report failures there; I imagine `Config`s will usually be hardcoded.

Reply:
Removed this commit from the branch.